lib/recurly/hosted-fields.js

```diff
@@ -96,17 +100,37 @@ export class HostedFields extends Emitter {
     this.initQueue = [];
   }
 
-  readyHandler (body) {
+  onReady (body) {
     const pos = this.initQueue.indexOf(body.type);
     if (~pos) this.initQueue.splice(pos, 1);
     if (this.initQueue.length === 0) {
-      this.off('hostedField:ready', this.readyHandler);
+      this.off('hostedField:ready', this.onReady);
       this.bus.send('hostedFields:ready');
       this.ready = true;
     }
     this.update(body);
   }
 
+  onTab (direction, message) {
+    const origin = this.getFieldByType(message.type);
+    let tabbableItems = tabbable(global.document.body);
+    let destination;
+
+    // Find the origin within the tabbable list, and focus in our intended direction
+    tabbableItems.find((el, i) => {
+      if (origin instanceof HostedField) {
+        if (origin.tabbingProxy === el) {
+          destination = direction === 'previous' ? tabbableItems[i - 1] : tabbableItems[i + 1];
+        }
+      } else {
+          destination = direction === 'previous' ? (tabbableItems[i - 1] || last(tabbableItems)) : (tabbableItems[i + 1] || tabbableItems[0]);
```

Comment:
is this logic path ever hit? `getFieldByType` will always return a `HostedField`, and this line references a function `last` that is never defined.